pkg/tests/tests.go

```diff
@@ -56,3 +53,27 @@ var tests = []helpers.TestOptions{
 		Handler:  handlers.TestStaticEndpoint,
 	},
 }
+
+func accessReviewTest() helpers.TestOptions {
```

Comment:
I would prefer this method return `[]byte` that way you would simply call 
``
 ..
 t.Body: `accessReviewReqBody()`
```

But since Marshalling can fail... this is OK :+1:

Reply:
Yes, ran into this I wanted also to only return the body, let me see what I can do to do it like that.